Containerfile

```diff
@@ -769,6 +769,7 @@ RUN rm -f /etc/profile.d/toolbox.sh && \
     systemctl disable tailscaled.service && \
     systemctl enable dev-hugepages1G.mount && \
     systemctl enable bazzite-snapper-setup && \
```

Comment:
what why is bazzite-snapper-setup enabled? That should not enable snapper right? @HikariKnight

Reply:
nope it should not enable any of the automatic snapshotting, it didnt on my testing

Reply:
Lets remove it anyway, it is causing issues

You can convert it to a ujust instead that also enables the service

Reply:
No need for ujust, the cleanup thing can be enabled from btrfs-assistant gui